openconfig_pyang/plugins/yangpath.py

```diff
@@ -175,19 +190,41 @@ def print_node(node, module, fd, prefix, ctx, level=0):
         print_children(node.i_children, module, fd, prefix, ctx, level)
 
 
-def get_pathstr(pathstr, config, ctx, level):
+def get_pathstr(pathstr, config, ctx, level, modname):
 
   if ctx.opts.print_plain or ctx.opts.relocate_output:
     return pathstr
 
   s = ''
-  if ctx.opts.print_depth:
-    s += '  [%d]' % level
-  else:
-    s += '     '
-  if config:
-    s += '  [%s] %s' % (config, pathstr)
+  if ctx.opts.prefix_module:
+    s += '%s:' % modname
+  if ctx.opts.sparse:
+    if ctx.opts.print_depth:
+      s += ' [%d]' % level
+    if ctx.opts.config_after:
+      if config:
+        s += ' %s [%s]' % (pathstr, config)
+      else:
+        s += ' %s' % (pathstr)
+    else:
+      if config:
+        s += ' [%s] %s' % (config, pathstr)
+      else:
+        s += ' %s' % (pathstr)
   else:
-    s += '       %s' % (pathstr)
+    if ctx.opts.print_depth:
```

Comment:
Should this block become a function that just takes an argument of the indentation to be applied (since it looks like lines 204-208 differ from this block only based on the indentation on line 223 and 228?